### ISSUE TITLE
Py.Test contrib does not work with pytest-bdd library.

### ISSUE THREAD

**Issue by proofit404** [dry-python/stories#135]

**Comment by atuchak** [comment]
My temporary workaround is to use "test_" prefix in the function name.
and pytest skip to preserve pytest from running the function as test case.

```
@pytest.mark.skip
@when('when')
def test_when2(ctx):
    Service().do_smth.run(arg=ctx.arg)  
```

**Comment by proofit404** [comment]
If you comment out [this line](https://github.com/dry-python/stories/blob/master/src/stories/contrib/pytest.py#L39), will it fix the issue?

**Comment by proofit404** [comment]
The fix was released in `0.10.1`.
